scipy/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
I'd feel much more comfortable if you add `set -euo pipefail`.
Next time someone edits this file they'll assume the new command they just added will fail the script...

Reply:
Done!  This also required some fancy `||...&&` work.